### PR TITLE
Linting and Cleanup

### PR DESCRIPTION
**Pull request by alexfikl**
Commits should be self-contained.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -88,20 +88,6 @@ jobs:
...
-                cd "$DOWNSTREAM_PROJECT"
-                echo "*** $DOWNSTREAM_PROJECT version: $(git rev-parse --short HEAD)"
-                edit_requirements_txt_for_downstream_in_subdir
-                export CONDA_ENVIRONMENT=.test-conda-env-py3.yml
-                # Avoid slow or complicated tests in downstream projects
-                export PYTEST_ADDOPTS="-k 'not (slowtest or octave or mpi)'"
-                build_py_project_in_conda_env
-                test_py_project
-
+                export PYTEST_ADDOPTS="-k 'not slowtest'"
```
Comment (alexfikl) [inline .github/workflows/ci.yml:91]: This was copied from `meshmode` which doesn't have the `not slowtest` part. Was that on purpose?

Comment (inducer) [reply .github/workflows/ci.yml:91]: Don't recall that being on purpose. :shrug: Happy to wait for it to cause a problem.

**Comment by inducer** [comment]
Thanks!

File: .github/workflows/ci.yml (hunk 1 above)
Comment (alexfikl) [reply .github/workflows/ci.yml:91]: Fair enough! I was thinking of some of those heavy 3D tests in pytential, but it's probably not that bad.

**Pull request merged**